tests/tests.yaml

```diff
@@ -28,8 +28,8 @@ answer_tests:
     - yt/frontends/athena/tests/test_outputs.py:test_blast
     - yt/frontends/athena/tests/test_outputs.py:test_stripping
 
-  local_athena_pp_004:
-    - yt/frontends/athena_pp/tests/test_outputs.py:test_disk
+  local_athena_pp_006:
```

Comment:
Just so I know -- why are we skipping 5, and can you put a comment in this file as to why we are deactivating `test_disk`?

Reply:
Skipping 5 because I made a mistake in incrementing the number--but I can make a comment here